ablySpec/RealtimeClientChannel.swift

```diff
@@ -526,6 +526,78 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6g
+                context("Identified clients with clientId") {
+
+                    // RTL6g1
+                    context("When publishing a Message with clientId set to null") {
+
+                        // RTL6g1a
+                        pending("should be unnecessary to set clientId of the Message before publishing") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            options.clientId = "client_string"
```

Comment:
@mattheworiordan Ok, I see. A little confusing because the `RTL6g` says _Identified clients with a clientId _... I thought the client needs  a `clientId` but the message is published without a `clientId` (_clientId attribute set to null_) like I did:

```
let message = ARTMessage(data: "message", name: nil)
expect(message.clientId).to(beNil())
```

I will fix the test.